documentation/docusaurus.config.ts

```diff
@@ -118,7 +118,7 @@ const config: Config = {
           items: [
             {
               label: "Discord",
-              href: "https://discord.gg/tdGPnnxf",
+              href: "https://discord.com/invite/s233GcZqFT",
```

Comment:
Was the link generated in a way it doesn't expire?

Reply:
Yes, it appears to be based on the blame and discord either allows 7 days or Never for expiration.

Reply:
I couldn't generate a link that never expires because I'm not an admin of the Discord Gophers discord. I've added one that an admin sent me :)